src/Form/CaptchaType.php

```diff
@@ -30,7 +30,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'hcaptcha_theme' => 'light',
             'recaptcha_theme' => 'light',
             'hcaptcha_public_key' => '',
-            'recaptcha_public_key' => '',
+            'recaptcha_public_key' => ''
```

Comment:
Let's keep the trailing comma 

```suggestion
            'recaptcha_public_key' => '',
```